lib/connect.c

```diff
@@ -1078,7 +1080,17 @@ static CURLcode singleipconnect(struct connectdata *conn,
         rc = connect(sockfd, &addr.sa_addr, addr.addrlen);
       }
 #endif /* HAVE_BUILTIN_AVAILABLE */
-#elif defined(MSG_FASTOPEN) /* Linux */
+#elif defined(TCP_FASTOPEN_CONNECT) /* Linux >= 4.11 */
+      int optval = 1;
+
+      if(setsockopt(sockfd, IPPROTO_TCP, TCP_FASTOPEN_CONNECT,
+            (void *)&optval, sizeof(optval)) < 0)
```

Comment:
column alignment should be used on continuation lines when possible 
~~~c
if(setsockopt(sockfd, IPPROTO_TCP, TCP_FASTOPEN_CONNECT,
              (void *)&optval, sizeof(optval)) < 0)
~~~

Reply:
Fixed.